### ISSUE TITLE
Yellow arrow pointing to off-screen items for inventory screen

### ISSUE THREAD

**Issue by The-Alchemist** [codecombat/codecombat#3458]
![codecombat-arrow in strange place](https://cloud.githubusercontent.com/assets/812287/13649732/0906e97c-e60d-11e5-92a4-9cb5153c0d2d.png)

I'm not sure what item it's expecting me to equip...

**Comment by judowalker** [comment]
I think something similar to this should work to bring the item into view before the yellow pointer targets the item:
``` js
var item = $("#unequipped").find(".should-equip");
var itemHeight = item.outerHeight(true);
var itemOffsetTop = item[0].offsetTop;
var itemOffsetBottom = itemOffsetTop + itemHeight;
var parent = item.parent();
var parentHeight = parent.height();
var parentScrollTop = parent.scrollTop();

if (itemOffsetBottom > parentHeight + parentScrollTop) {
	parent.scrollTop(itemOffsetBottom - parentHeight);
}
```
I guess I could try submitting a pull request but I've never used CoffeeScript.

**Comment by nwinter** [comment]
Thanks @judowalker, this code is great! I just implemented in 1933caf04ad380e49963f964d18e5ed09a5a082e, so this should finally be good to go as soon as that's merged in.
